tests/TenantManagerTest.php

```diff
@@ -235,6 +237,28 @@ public function automatic_migrations_work()
         $tenant2 = Tenant::create(['bar.localhost']);
         tenancy()->initialize($tenant2);
         $this->assertTrue(\Schema::hasTable('users'));
+        $this->assertCount(0, \DB::select('select * from users'));
```

Comment:
I guess this could be changed to `assertSame` too?

Reply:
Oh, yeah, nice catch.

Reply:
Maybe it's not even needed?

Reply:
Actually, on second thought, I'd remove the assertion. Just assert that the table doesn't exist. Making queries to nonexistent tables is confusing and I'm surprised it doesn't throw an exception.

Reply:
Removal makes sense.

There's no exception because the table does exist at that point. Just empty

Reply:
Ah, I misread the line above, thought it's `assertFalse`. But yeah, I'd remove it anyway because we're testing migrations at this point.

Reply:
Done